### DISCUSSION TITLE
what the purpose of --service?

### DISCUSSION THREAD

**Discussion by congzhangzh** [rustdesk/rustdesk#742] [Q&A]
as I understand, the ui access need be done after login, so what's the purpose of run as a global service?

**Comment by rustdesk** [comment]
for before login

**Reply by congzhangzh** [reply]
En, I do not understand, what this service can provide? Support connect to or not? But what to show,  As no ui to remote share

**Reply by WhiteAls** [reply]
regarding windows os, running as a system service gives you the ability to connect to the login screen and interact with uac
